gilded-rose-llc-frontend/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Comment:
Does this add rules such as no explicit any, no unused variables?

Reply:
Thank you for pointing this out, another area that I would have overlooked. No, this does not include those rules. This is the standard NextJS configuration but realizing now that this is also typescript so we need to add the linter config rules for that. 

According to that doc we are missing the prettier extension on that file as well. I will make the proper adjustments so that dependency is added and its not just configured in my IDE.

cc. @jonmanahan -> https://paulintrognon.fr/blog/typescript-prettier-eslint-next-js 👀

Reply:
This is addressed in [dc7be1a](https://github.com/jramnani/GildedRoseJavaLLC/pull/27/commits/dc7be1aa795f739f2e49342b26c149f3121f92ea)

- eslint was updated to include [@typescript-eslint plugin](https://github.com/typescript-eslint/typescript-eslint) with rulesets
- prettier added as dependency with prettierrc.json config file with basic setup
- README updated to include prettier

Thanks again for pointing this out @mrfishball, this was some good information I looked into that I probably wouldn't have otherwise

Reply:
😄